### ISSUE TITLE
predbat.plan_html exceed maximum size of 16384 bytes

### ISSUE THREAD

**Issue by dbrb2** [springfall2008/batpred#438]
This doesn't seem to cause any real world problems, but seemed worth mentioning - as has been cropping up in the logs a lot: 

```
Logger: homeassistant.components.recorder.db_schema
Source: components/recorder/db_schema.py:582
Integration: Recorder (documentation, issues)
First occurred: 12:07:19 (31 occurrences)
Last logged: 13:50:19

State attributes for predbat.plan_html exceed maximum size of 16384 bytes. This can cause database performance issues; Attributes will not be stored
```

**Comment by gcoan** [comment]
Have a look at the faq’s, I’ve added details of this issue last week, and how to suppress it

https://springfall2008.github.io/batpred/faq/

Geoffrey

**Comment by dbrb2** [comment]
Ah thanks
That didn't quite work for me - I had to specify which logger to filter:

logger:
  default: info
  filters:

*    homeassistant.components.recorder.db_schema:*      - "State attributes
for predbat.plan_html exceed maximum size of 16384 bytes. This can cause
database performance issues; Attributes will not be stored"

Seems to work!

**Comment by gcoan** [comment]
Thanks @dbrb2 that was my fault, cut and paste error when copying the code from my configuration.yaml across into predbat

It was part of a larger filter I had to also remove similar octopus energy warnings, vis:
    homeassistant.components.recorder.db_schema: 
      - "State attributes for sensor.octopus_energy_electricity_xxxx_export_current_rate exceed maximum size of 16384 bytes. This can cause database performance issues; Attributes will not be stored"
      - "State attributes for sensor.octopus_energy_electricity_xxxx_current_rate exceed maximum size of 16384 bytes. This can cause database performance issues; Attributes will not be stored"
      - "State attributes for predbat.plan_html exceed maximum size of 16384 bytes. This can cause database performance issues; Attributes will not be stored"
    homeassistant.helpers.entity:
      - "Update of sensor.octopus_energy_electricity_xxxx_export_previous_accumulative_cost is taking over 10 seconds"
      - "Update of sensor.octopus_energy_electricity_xxxx_previous_accumulative_cost is taking over 10 seconds"

and I accidentally deleted the logger line.

Corrected in PR #439

**Comment by gcoan** [comment]
@springfall2008 think this can be closed now?
